### PR TITLE
Issue#214

### PR DESCRIPTION
**Pull request by charlie0605**
Backdrop is presented on slick
image

Next button scrolls four images at a time when it is the last image it will just scroll once.
When clicked once: 
image
When clicked again:
image

The image is now drawn on click whereas before takes a few clicks on the previous version.

Thumbnail size is adjusted according to the screen size.
image
image

Thumbnail is aligned vertically.

### CODE REVIEW

File: app/assets/javascripts/channels/backdrops.coffee [hunk 1]
```diff
@@ -1,5 +1,9 @@
 display = (data) ->
   backdrop = new Image(data.backdrop_id)
+
+  backdrop.onload = ->
+    App.drawFrame();
+
```
Comment (thongteav) [inline app/assets/javascripts/channels/backdrops.coffee]: Is this to do with the clicking function of images?

File: app/assets/stylesheets/theatre.scss [hunk 2]
```diff
@@ -163,3 +166,32 @@ div.message > div.content.shout {
...
+
+.slick-next {
+  right: -5px;
+  z-index: 10;
+}
+
+.slick-prev {
+  left: -10px;
+  z-index: 10;
+}
```
Comment (thongteav) [inline app/assets/stylesheets/theatre.scss]: Is there a reason for having different offset values for next and previous buttons?

File: app/assets/stylesheets/theatre.scss [hunk 3]
```diff
@@ -163,3 +166,32 @@ div.message > div.content.shout {
   font-weight: bold;
   text-transform: uppercase;
 }
+
+.scroller {
+  div {
+    font-size: 30px;
+  }
+}
```
Comment (thongteav) [inline app/assets/stylesheets/theatre.scss]: I don't think there's a need to adjust the font size in here.

File: app/assets/javascripts/channels/backdrops.coffee (hunk 1 above)
Comment (charlie0605) [reply app/assets/javascripts/channels/backdrops.coffee]: yes

File: app/assets/stylesheets/theatre.scss (hunk 3 above)
Comment (charlie0605) [reply app/assets/stylesheets/theatre.scss]: I will remove it, don't think this has any effect

File: app/assets/stylesheets/theatre.scss (hunk 2 above)
Comment (charlie0605) [reply app/assets/stylesheets/theatre.scss]: left button needs to be moved more to the right compared to the right button, it doesn't need to move to the left as much

**Pull request merged**